src/agent/Makefile

```diff
@@ -57,12 +57,26 @@ INIT := no
 # Path to systemd unit directory if installed as not init.
 UNIT_DIR := /usr/lib/systemd/system
 
+GENERATED_CODE = src/version.rs
+
+AGENT_NAME=$(TARGET)
+API_VERSION=0.0.1
+AGENT_VERSION=$(VERSION)
+
+GENERATED_REPLACEMENTS=	BINDIR \
```

Comment:
A couple of nits:

- It looks a little odd with the whitspace. How about:

```make
GENERATED_REPLACEMENTS= \
                      AGENT_NAME \
                      AGENT_VERSION \
                      BINDIR \
                         :
```

- To assuage my OCD, could you sort the list alphabetically? :smile:

Reply:
Pushed a new version that fixes this and the `agent:` prefix.

I also replaced `[@]foo[@]` in `sed` with `@foo@` if you don't mind. Let me know otherwise.